### PR TITLE
Blog app - Views

### PR DESCRIPTION
**Pull request by VaheStepanyan100**
- Implement the design from the sneak peek wireframes.

### CODE REVIEW

File: app/views/users/index.html.erb [hunk 1]
```diff
@@ -1 +1,13 @@
...
+  <% @users.each do |user| %>
+  <li>
+    <a href="<%= user_posts_path(user) %>">
+      <div class="user-box">
+        <%= render partial: 'shared/user_photo', locals: { user: user } %>
+        <%= render partial: 'shared/user_details', locals: { user: user } %>
+     </div>
+  </a>
+  </li>
+  <% end %>
```
Comment (Gambit142) [inline app/views/users/index.html.erb]: - [ ] Based on the requirements, when I click on a user on the `Users#Index` page, it should lead me to the `Users#Show` page but your application leads to the `Posts#Index` page. Kindly correct this. 👍 😎

File: app/views/users/index.html.erb [hunk 2]
```diff
@@ -1 +1,13 @@
...
\ No newline at end of file
+<h1>Users</h1>
+<ul>
+  <% @users.each do |user| %>
+  <li>
+    <a href="<%= user_posts_path(user) %>">
+      <div class="user-box">
+        <%= render partial: 'shared/user_photo', locals: { user: user } %>
+        <%= render partial: 'shared/user_details', locals: { user: user } %>
+     </div>
```
Comment (Gambit142) [inline app/views/users/index.html.erb:9]: - [ ] Great work using partials to render commonly shared features across different pages. Great work 👏 💪

**Review by Gambit142 — CHANGES_REQUESTED** [review]
Hi @VaheStepanyan100 👋, 

Good job so far!

## Highlights
- Setup linters correctly ✔️ 
- Created view files for the `Users` model ✔️ 
- Created view files for the `Posts` model ✔️ 

You still need to work on some issues to go to the next project, but you are almost there!

## Required Changes ♻️
_Kindly check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend considering them as they can improve your code._

You can also consider the following:

N/A

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### Please tag me in your question so I can receive the notification. You can tag me by typing @Gambit142

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Comment by VaheStepanyan100** [comment]
Thank you @Gambit142  for the review! Your input is much appreciated! 👍 😊

**Review by prolajumokeoni — APPROVED** [review]
Hi @VaheStepanyan100 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
## Highlights
 - Path to user's show page updated ✅  

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by VaheStepanyan100** [comment]
Thank you so much @Gambit142

**Pull request merged**